clang/test/CodeGenCXX/visibility-dllstorageclass.cpp

```diff
@@ -79,6 +107,8 @@ extern void __declspec(dllimport) imported_e();
 // EXPLICIT-DAG: declare hidden void @_Z10imported_ev()
 // ALL_DEFAULT-DAG: declare void @_Z1ev()
 // ALL_DEFAULT-DAG: declare void @_Z10imported_ev()
+// ALL_KEEP-DAG: declare hidden void @_Z1ev()
+// ALL_KEEP-DAG: void @_Z10imported_ev()
```

Comment:
Oh, this line looks like it would allow any visibility (missing `declare` and whatever might be between that and the `void`)